internal/pkg/deploy/env.go

```diff
@@ -13,7 +13,7 @@ const (
 	// LegacyEnvTemplateVersion is the version associated with the environment template before we started versioning.
 	LegacyEnvTemplateVersion = "v0.0.0"
 	// LatestEnvTemplateVersion is the latest version number available for environment templates.
-	LatestEnvTemplateVersion = "v1.7.0"
+	LatestEnvTemplateVersion = "v1.7.1"
```

Comment:
no need to update it 😎 from the point of view of the customers they are still till on v1.6.2 since v1.7.0 of the template hasn't been released yet

Reply:
oh right right